### ISSUE TITLE
Case-insensitive filenames generated even when --case-insensitive-filenames not specified

### ISSUE THREAD

**Issue by andrewlalis** [adamdruppe/adrdox#53]
I'm trying to use adrdox to automatically generate documentation for my [slf4d](https://github.com/andrewlalis/slf4d) module, but it looks like it's generating filenames where an underscore precedes every capitalized letter, so I end up with things like `slf4d.default_provider.factory._Default_Logger_Factory.set_Root_Level.html` instead of `slf4d.default_provider.factory.DefaultLoggerFactory.setRootLevel.html`.

My script is located in that repository, under `docs/build-ddoc.d`. Essentially, it is as follows:
```
./doc2 -i -o ./src/.vuepress/public/ddoc ../source
```

**Comment by adamdruppe** [comment]
yeah, on windows it defaults to true and on linux it defaults to false. it is so it doesn't overwrite things. The links should all be adjusted properly internally so it all works though....

**Comment by andrewlalis** [comment]
Ah, good to know. This happened on MacOS, by the way.

**Comment by adamdruppe** [comment]
oh yeah, mac i looked up a long time ago and figured better to keep the insensitive there too.
